### ISSUE TITLE
Add option to remove fold markers

### ISSUE THREAD

**Issue by a-sk** [jeffbuttars/Vimdown#1]
Hi, nice tool ! Thank you for your work.

Can you add an option to delete vim fold markers like `{{{` and `}}}`

**Comment by jeffbuttars** [comment]
Yes. That shouldn't be too difficult. I'll be on holiday for a while, so I may not get to it for a bit.

**Comment by a-sk** [comment]
In this case, maybe I'll come up with a pull request if it's not so difficult.

**Comment by jeffbuttars** [comment]
That would be excellent.
FYI. My next step with vimdown was to make it more versatile. It uses a hacked together custom lexer/parser. I plan on implementing a better parser and then making vimdown into a more general tool that can convert more than just vim files. I think it would be a small step to be able to convert other simple formats, like bash for example.

**Comment by a-sk** [comment]
Hm, nice, **nice** idea.
Sometimes I can be smashed out how incredibly useful a simple idea could be.
_ps:I will defiantly take a look at the code and hack it a bit._
